targets/HUDL/main.cpp

```diff
@@ -225,7 +168,34 @@ int main() {
         COTmrService(&canNode.Tmr);
         // Handle executing timer events that have elapsed
         COTmrProcess(&canNode.Tmr);
-        // Wait for new data to come in
-        //        time::wait(10);
     }
 }
+
```

Comment:
Why did you move these down here? I see why you might argue this makes the code a bit cleaner, but if you want to modify how we handle this, we should change every board to match. If this is something you want, let's talk more

Reply:
I think I moved them down here because I was doing a lot of up and down scrolling in a main file and just kept getting lost in the weeds there. I do want to talk about that though because I am really not sure what is better for them. They kind of disappear when they are at the bottom, which has once led me to think they were not in there at all. I think this can be addressed in the CANOpen upgrade, so we should talk about this.

Reply:
Grab me on Monday, so we can discuss this